MemberDetailsFrontEnd.php

```diff
@@ -0,0 +1,660 @@
+  html += '<td><select name="Category[]" class="form-control Category" data-badgeid="'+count+'"><option value="">Select Category</option><?php echo fill_select_box($connect); ?></select></td>';
+  html += '<td><select name="Badge[]" class="form-control Badges" id="Badge'+count+'" ><option value="">Select Badge</option></select></td>';
+  html += '<td><select name="Level[]" class="form-control Level" id="Level'+count+'"><option value="">Select Level</option></select></td>';        
+  html += '<td><input type="text" name="Quantity[]" class="form-control Quantity" id="Quantity"></td>';  
+  html += '<td style="width:9%; text-align:center"><input type="submit" name="submit" class="btn btn-success btn-sm glyphicon glyphicon-plus"value= "+"/ style="margin-top: 3px"></td>'; 
+  html += '<td style="width:9%; text-align:center"><button type="button" name="remove" class="btn btn-danger btn-sm remove glyphicon glyphicon-minus"value= "-"/  style="margin-top: 3px"></button></td></tr>'; 
+  // html += '<td><button type="button" name="remove" class="btn btn-danger btn-xs remove"><span class="glyphicon glyphicon-minus"></span></button></td>';
+  $('tbody').append(html);
+});
+ 
+//remove row when add badge button clicked
+$(document).on('click', '.remove', function(){
+    $(this).closest('tr').remove();
+  });
+
+//Badge select box based on category
+  $(document).on('change', '.Category', function(){
+    var CategoryID = $(this).val();
+    var badgeid = $(this).data('badgeid');
+    
+        $.ajax({
+            type:'POST',
+            url:'SelectBadge.php',
+            data: "CategoryID="+CategoryID,
+            dataType:"json",
+            success:function(data){
+              var html = '';
+              var html = '<option value="">Select Badges</option>';
+              for(var count = 0; count < data.length; count++)
+              {
+                html += '<option value="'+data[count].id+'">'+data[count].name+'</option>';
+              }                  
+              $('#Badge'+badgeid).html(html); 
+                
+            }
+        });       
+  });
+
+//Level select box based on category
+$(document).on('change', '.Category', function(){
+    var CategoryID = $(this).val();
+    var badgeid = $(this).data('badgeid');
+    
+        $.ajax({
+            type:'POST',
+            url:'SelectLevel.php',
+            data: "CategoryID="+CategoryID,
+            dataType:"json",
+            success:function(data){
+              var html = '';
+              var html = '<option value="">Select Level</option>';
+              for(var count = 0; count < data.length; count++)
+              {
+                html += '<option value="'+data[count].id+'">'+data[count].name+'</option>';
+              }                  
+              $('#Level'+badgeid).html(html); 
+                
+            }
+        });       
+});
+    
+
+ $('#insert_form_temp').on('submit', function(event){
+event.preventDefault();
+    var error = '';

```

Comment:
get into the habit of using `const` and `let` instead of `var` const variables never gets reassigned, whereas let can be.